pkg/cmd/open.go

```diff
@@ -80,10 +80,23 @@ var openGithubCmd = &cobra.Command{
 	},
 }
 
+var openDocsCmd = &cobra.Command{
+	Use:   "docs",
+	Short: "Open the Doppler documentation home page",
+	Args:  cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		err := open.Run("https://dashboard.doppler.com/workplace/docs")
```

Comment:
For good measure, should this be based on the `DashboardHost`?

Reply:
Should we just do `https://docs.doppler.com/`? In case someone is not logged in?